.storybook/recipes/BaselineCard/BaselineCard.tsx

```diff
@@ -2,9 +2,7 @@ import clsx from 'clsx';
 import React, { MutableRefObject, ReactNode, useEffect, useRef } from 'react';
 import styles from './BaselineCard.module.css';
 
-import { Card, CardBody, CardFooter, CardHeader, Tag } from '../../../src';
-
-import { Variant } from '../../../src/components/Tag/Tag';
```

Comment:
Should we import a `Variant` type from `Score` instead?

Reply:
Good question, I guess I will since Tag does and wouldn't hurt to